tutorials/v7/box.py

```diff
@@ -0,0 +1,41 @@
+## \file
+## \ingroup tutorial_v7
+##
+## This ROOT 7 example demonstrates how to create a ROOT 7 canvas (RCanvas) and
+## draw ROOT 7 boxes in it (RBox). It generates a set of boxes using the
+## "normal" coordinates' system.
+## Run macro with python3 -i box.py command to get interactive canvas
+##
+## \macro_image (rcanvas_js)
+## \macro_code
+##
+## \date 2021-06-15
+## \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
+## is welcome!
+## \author Sergey Linev
+
+import ROOT
```

Comment:
Can we do without this line?

Reply:
No - python fails in next string with `from ROOT.Experimental import RCanvas, RBox, RPadPos, RColor`

Reply:
No:
```
ModuleNotFoundError: No module named 'ROOT.Experimental'; 'ROOT' is not a package
```